osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/utils/StacktraceUtils.java

```diff
@@ -0,0 +1,11 @@
+package org.opensmartgridplatform.shared.utils;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/** Convenience methods dealing with stacktraces. */
+public class StacktraceUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1118)